obspy/signal/tests/test_spectral_estimation.py

```diff
@@ -565,6 +565,22 @@ def test_earthquake_models(self):
             caption = 'M%.1f\n%dkm' % (magnitude, distance)
             self.assertEqual(ax.texts[0].get_text(), caption)
 
+    def test_ppsd_infrasound(self):
```

Comment:
This test is only of limited benefit right now. It basically just assures that something is done and no error pops up. There's two ways here. Either set up an image test, but that should really be avoided when possible, as it's a huge maintenance burden. So, ideally what you want to do is simply inspect the artists in the figure and do some assertions on those.

See the `test_earthquake_models` test case in `obspy/signal/tests/test_spectral_estimation.py` for an example:

https://github.com/obspy/obspy/blob/a8750f46132d937daec3a41ed6d5f38478a4ecc0/obspy/signal/tests/test_spectral_estimation.py#L544-L566

Also what @heavelock said, the test data should be at most the length such that 2 or 3 PSD pieces get processed. Adding more data does not help the test in any way and just burdens our repository (and git history) and slows down the testing.

(eventually that 30MB file needs to get rebased out of the PR history, so when you `git remove` that file, best do it in a single commit that can then easily be squashed on top of the commit that added the file in the first place)

Reply:
Thank you for the advice. I have been attempting to implement it, but I have managed to mess something up in the test, and I'm a little stuck. When I run the obspy-runtests, I get
`>>> Cannot import test suite for module obspy.signal due to:
------------------------------------------------------------
  File "/Users/ken/opt/anaconda3/envs/obspy_dev/bin/obspy-runtests", line 33, in <module>
    sys.exit(load_entry_point('obspy', 'console_scripts', 'obspy-runtests')())
  File "/Users/ken/obspy/obspy/scripts/runtests.py", line 744, in main
    errors = run(argv, interactive)
  File "/Users/ken/obspy/obspy/scripts/runtests.py", line 712, in run
    pr_url=args.pr_url)
  File "/Users/ken/obspy/obspy/scripts/runtests.py", line 537, in run_tests
    suites, status, import_failures = _get_suites(verbosity, tests)
  File "/Users/ken/obspy/obspy/scripts/runtests.py", line 161, in _get_suites
    suite.append(ut.loadTestsFromName(test, None))
  File "/Users/ken/opt/anaconda3/envs/obspy_dev/lib/python3.6/unittest/loader.py", line 204, in loadTestsFromName
    test = obj()
  File "/Users/ken/obspy/obspy/signal/tests/__init__.py", line 12, in suite
    add_doctests(suite, MODULE_NAME)
  File "/Users/ken/obspy/obspy/core/util/testing.py", line 115, in add_doctests
    fromlist=["obspy"])

ModuleNotFoundError: No module named 'obspy.signal.'`

Reply:
I don't know what changed, but the test runs fine now.